src/data/faq.json

```diff
@@ -101,6 +101,166 @@
                     }
                 ]
             },
+            {
+                "title": "Vaccination Certificate",
+                "id": "vaccination_certificate",
+                "active": true,
+                "accordion": [
+                    {
+                        "title": "Questions and answers about the vaccination certificate",
+                        "anchor": "vaccination_certificate_index",
+                        "active": true,
+                        "textblock": [
+                            "<ul>",
+                            "<li><a href='#vaccination_certificate_multiple_scan_possible' target='_self' rel='noopener noreferrer'>Can I save my vaccination certificate on multiple smartphones, e.g., on my private and business phone?</a></li><li><a href='#vaccination_certificate_multiple_scan_possible' target='_self' rel='noopener noreferrer'>How can I transfer the vaccination certificate when changing device?</a></li><li><a href='#vaccination_certificate_multiple_scan_possible' target='_self' rel='noopener noreferrer'>What do I do if I lose my phone?</a></li><li><a href='#vaccination_certificate_multiple_scan_possible' target='_self' rel='noopener noreferrer'>Will the vaccination certificate be included in my backup?</a></li><li><a href='#vaccination_certificate_multiple_scan_possible' target='_self' rel='noopener noreferrer'>Can I manage the certificate in the Corona-Warn-App and the CovPass-App at the same time, or do I have to choose one of the two apps?</a></li>",
+                            "</ul>"
+                        ]
+                    },
+                    {
+                        "title": "What is the digital certificate of vaccination?",
+                        "anchor": "vaccination_certificate_what",
+                        "active": true,
+                        "textblock": [
+                            "Starting with version 2.3 of the Corona-Warn-App (CWA), you can add your digital vaccination certificate in the CWA and thus securely and digitally prove your Corona vaccination status via 2D barcode (QR code). The CWA indicates the complete vaccination protection 14 days after the last required vaccination. The digital vaccination certificate is an additional way to document vaccinations. Thereby you are able to digitally store information such as vaccination time and vaccine in a personalised manner on their smartphones. The use of digital certificate of vaccination in the CWA is voluntary. You can also prove your Corona vaccination in a different way."

```

Comment:
"Thereby you are able to digitally store information such as vaccination time and vaccine in a personalised manner on their smartphones."
Generally we use United States spelling, so "personalised" => "personalized".
"their smartphones." => "your smartphone" (singular) or  "your smartphones" (plural)

Reply:
fixed